Files/co_021290.csv

```diff
@@ -1,5 +1,5 @@
-text00#0,"""進行状況のセーブ/ロードについて"""
-text01#0,"""このクエストは、進行状況が記録されており<br>クエストを破棄した場合でも、一定の場所から<br>続きを遊ぶことが可能となっています。"""
-text02#0,"""進行状況のセーブについて<br><br>クエストの進行状況がセーブされる際には<br>「クエストの進行を記録しました」というメッセージが表示されます。<br>以後、その場所からクエストを遊ぶことが可能になります。"""
-text03#0,"""進行状況のロードについて<br><br>クエストを受託後、進行状況がセーブされている状態で<br>キャンプシップのテレプールに入ると、確認ウィンドウが表示されます。"""
-text04#0,"""このとき「続きからプレイする」を決定すると<br>前回の進行状況の続きからプレイすることができます。<br>※続きからプレイしても、記録されたデータは削除されません。"""
+text00#0,"""About Saving / Loading mission progress"""
```

Comment:
If this line is used in the context of a window title or a sub-section title, it should have title-case capitalization.